velox/duckdb/conversion/DuckParser.h

```diff
@@ -40,4 +41,39 @@ std::shared_ptr<const core::IExpr> parseExpr(const std::string& exprString);
 std::pair<std::shared_ptr<const core::IExpr>, core::SortOrder> parseOrderByExpr(
     const std::string& exprString);
 
+// Parses a WINDOW function SQL string using DuckDB's internal postgresql-based
```

Comment:
Use `///` for headers

Reply:
Leaving it // to match other comments in this file.

Reply:
Leaving it // to match other comments in this file.